pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -79,6 +79,9 @@ public static String toJsonMapStr(@Nullable Map map)
   @ScalarFunction(nullableParameters = true)
   public static String jsonFormat(Object object)
       throws JsonProcessingException {
+    if (object == null) {
```

Comment:
This is intentionally added in #8382. If we want to return null, removing the `nullableParameters = true` will achieve that. Any specific reason why you don't want string `'null'`?

Reply:
Changed